packages/lib/gpcircuits/test/tuple.spec.ts

```diff
@@ -1,54 +1,68 @@
 import { WitnessTester } from "circomkit";
 import "mocha";
 import {
+  tupleHasher,
   TupleHasherInputNamesType,
-  TupleHasherInputs,
-  TupleHasherOutputNamesType,
-  TupleHasherOutputs
+  TupleHasherOutputNamesType
 } from "../src";
 import { circomkit } from "./common";
 
 describe("tuple.TupleHasher should work", function () {
   // Circuit compilation sometimes takes more than the default timeout of 2s.
-  let circuit: WitnessTester<
-    TupleHasherInputNamesType,
-    TupleHasherOutputNamesType
-  >;
+  const circuit: (
```

Comment:
Particularly given the need to duplicate the types, I think this would probably be clearer as a `function` declaration rather than a `const`

Reply:
Ah, the type duplication was unnecessary as it turns out. I had it in there back when the definition was split and forgot to get rid of it.